app/models/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  def thumbnail
+    return unless @record['pnx']['addata']['isbn']
+
+    # A record can have multiple ISBNs, so we are assuming here that
+    # the thumbnail URL can be constructed from the first occurrence
+    isbn = @record['pnx']['addata']['isbn'].first
+    [ENV.fetch('SYNDETICS_PRIMO_URL', nil), '&isbn=', isbn, '/sc.jpg'].join
```

Comment:
This brings up an interesting question... does our license allow us to use those thumbnails for metadata coming from different systems. My assumption is that comes along for usage with Primo/Alma and not "anything we want" but I'm not sure.

Reply:
I'd suggest we use https://openlibrary.org/dev/docs/api/covers as our initial implementation (and they say they appreciate a link back which we could find a place to include on an "about" page (i.e. they don't expect something intrusive).

Reply:
This was actually included by accident in the latest commit. Something I was starting to think about. 🙂 

You raise a good point, though... if we want thumbnails, we may need to include them in another way.